easy_music_generator/preprocessor/chord_distribution.py

```diff
@@ -51,7 +63,9 @@ def get_chord_probabilities(dictionary):
             dictionary[key] /= total
         return dictionary
 
+
 class NoChordsFoundException(Exception):
-    def __init__(self, message="The provided MIDI or MusicXML files do not contain any chords."):
+    def __init__(self, message="The provided MIDI or MusicXML "
+                               "files do not contain any chords."):
```

Comment:
Does this string message work? I believe this was broken down to follow PEP-8 for the number of characters in one line. 
I would recommend doing the following: 

  def __init__(self, 
                       message="The provided MIDI or MusicXML files do not contain any chords."):

Reply:
I think both the string cutting and removing init's underscores don't work here, so I'm ignoring PEP8 for this one